src/Service/SwaggerService.php

```diff
@@ -42,7 +42,7 @@ public function getViewName(): string
         return $this->viewName;
     }
 
-    public function fetch(array $annotationPaths): OpenApi
+    public function fetch(array $annotationPaths): ?OpenApi
```

Comment:
Should we take into account [here](https://github.com/yiisoft/yii-swagger/blob/ff203e5cd384fbcffd9b2c0c49704cfe2aaa8489/src/Middleware/SwaggerJson.php#L38) that `fetch()` returns nullable value?